Change_Log/Trigger_Teacher.sql

```diff
@@ -0,0 +1,99 @@
+ IF OBJECT_ID('Trigger_Teacher') IS NOT NULL  
+    DROP TRIGGER Trigger_Teacher;  
+GO
+CREATE TRIGGER Trigger_Teacher 
+ON [Teacher]
+AFTER INSERT, DELETE, UPDATE AS
+BEGIN
+	DECLARE @ChangeOperationType char
+	DECLARE @TrackedEntityId tinyint = 2
+	DECLARE @TrackedEntityRecordId bigint
+	DECLARE @Description varchar(max)
+	DECLARE @ChangeDateUTC datetime2
+	DECLARE @T_ChangeHistoryId varchar(max)
+
+	DECLARE @Previous_TeacherId bigint
+	DECLARE @Previous_Name varchar(20)
+	DECLARE @Previous_LevelId char(1)
+	DECLARE @Previous_Salary decimal(10,2)
+	DECLARE @Previous_AdmitionDate date
+
+	DECLARE @New_TeacherId bigint
+	DECLARE @New_Name varchar(20)
+	DECLARE @New_LevelId char(1)
+	DECLARE @New_Salary decimal(10,2)
+	DECLARE @New_AdmitionDate date
+
+	IF (
+		EXISTS(SELECT 1 FROM INSERTED)
+		AND EXISTS (SELECT 1 FROM DELETED)
+		AND NOT EXISTS
+			(	
+			SELECT * FROM INSERTED
+			INTERSECT
+			SELECT * FROM DELETED
+			)
+		)
+	BEGIN     
+		SELECT @TrackedEntityRecordId = TeacherId
+		FROM INSERTED
+		SET @ChangeOperationType = 'U'
+		SET @Description = 'The teacher ' + CAST(@TrackedEntityRecordId AS VARCHAR(MAX)) + ' was updated'
```

Comment:
Simplificar a lógica para CONCAT('The teacher ', @TrackedEntityRecordId, ' was updated')